frontend/src/router.ts

```diff
@@ -1,11 +1,14 @@
 import { createRouter, createWebHistory } from "vue-router";
 
 import ArticleView from "@/views/ArticleView.vue";
-import ClientView from "@/views/ClientView.vue";
+import ClientsView from "@/views/ClientsView.vue";
 import DocumentView from "@/views/DocumentView.vue";
+import ClientView from "@/views/ClientView.vue";
+
 
 export const ROUTES = {
   CLIENTS: { name: "clients", path: "/", label: "Kunden" },
```

Comment:
Looking at this, could we actually give the URL of /clients to the clients page and have a second URL or redirect to that one?